scss/mixin.scss

```diff
@@ -43,3 +43,30 @@
     }
   }
 }
+
+@mixin homepage-search-dropdown( $border-color, $home-dropdown ) {
```

Comment:
I created this mixin to differentiate between statement form elements on the homepage and regular ones throughout the site. Should I rename it?

Reply:
comment about the name of this mixin

- Let's remove `homepage` to make the name more generic (there aren't multiple versions of "search dropdown" style so we don't need to make the mixin name so specific)
- Can we rename it to `search-input-and-dropdown`? I know the name is loooong but `search-dropdown` makes me think that this mixin is for the help type dropdown only

Reply:
Cool cool, sounds good :)